engine/movie/intro.asm

```diff
@@ -462,5 +463,4 @@ FightIntroFrontMon3:
 ENDC
 
 FightIntroFrontMonEnd:
-
```

Comment:
This blank line represents separation between two things which are not strictly connected.